apps/aehttp/test/aehttp_ws_test_utils.erl

```diff
@@ -95,7 +95,7 @@ wait_for_connect(Pid) ->
         timeout ->
             case process_info(Pid) of
                 undefined -> {error, rejected};
-                _ -> {error, still_connecting, Pid}
+                _ -> {error, {still_connecting, Pid}}
```

Comment:
L3956 in aehttp_integration_SUITE also needs changing then...

Reply:
Seems confirmed by [CI](https://circleci.com/gh/aeternity/epoch/28355):
```
%%% aehttp_integration_SUITE ==> all.websocket.ws_refused_on_limit_reached: FAILED
%%% aehttp_integration_SUITE ==> {{badmatch,{error,{still_connecting,<0.1989.1>}}},
 [{aehttp_integration_SUITE,'-ws_refused_on_limit_reached/1-fun-1-',2,
                            [{file,"/home/builder/epoch/_build/test/lib/aehttp/test/aehttp_integration_SUITE.erl"},
                             {line,3956}]},
  {lists,map,2,[{file,"lists.erl"},{line,1239}]},
  {aehttp_integration_SUITE,ws_refused_on_limit_reached,1,
                            [{file,"/home/builder/epoch/_build/test/lib/aehttp/test/aehttp_integration_SUITE.erl"},
                             {line,3952}]},
  {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1546}]},
  {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1062}]},
  {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,994}]}]}
```